ase/src/main/java/com/ase/ase/services/DownloadScheduleService.java

```diff
@@ -0,0 +1,44 @@
+package com.ase.ase.services;
+
+import com.ase.ase.activemq.MessagingService;
+import com.ase.ase.activemq.UpdateDataMessage;
+import org.springframework.stereotype.Service;
+
+import java.util.Calendar;
+import java.util.Timer;
+import java.util.TimerTask;
+
+@Service
+public class DownloadScheduleService {
+
+    private void repeatPushingDataForDemo(MessagingService messagingService, DownloadService downloadService) {
```

Comment:
This method should be renamed. It does not push demo data anymore but downloads the data then notifies the frontend.